apps/src/javalab/CommitDialog.jsx

```diff
@@ -172,7 +172,8 @@ const styles = {
     borderBottom: `1px solid ${color.lightest_gray}`
   },
   fileLabel: {
-    flexGrow: 2
+    flexGrow: 2,
+    color: color.default_text
```

Comment:
Do you have a screenshot you could add?

Reply:
Added

Reply:
And what did it look like before?

Reply:
Ah yep that's in the JIRA: https://codedotorg.atlassian.net/browse/CSA-528